### PR TITLE
Little annoying things

### PR DESCRIPTION
**Pull request by legomannetje**
In the config, the SD speeds were not together. So I enabled the 5Mhz, and was confused it didn't work.

This way its more clear

### CODE REVIEW

**Comment by koriwi** [comment]
maybe move the lines up to where the 16mhz line was. It should be higher up in my opinion because this option will be changed the most.

**Comment by legomannetje** [comment]
The whole settings needs a rework I guess. Most people won't changes the pins. So make a basic and advanced part?

**Comment by koriwi** [comment]
good idea! you should create an issue for that. could you still make the requested changes for this pr?

**Comment by legomannetje** [comment]
Concider it done

**Comment by legomannetje** [comment]
Done

File: settings.hpp [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #define CONFIG_NAME_TEMP "config.tmp"
 #define CONFIG_NAME_OLD "config.bin.old"
 #define LONG_PRESS_DURATION 250
-#define TIMEOUT_TIME 5L * 60L // timeout in seconds
+#define TIMEOUT_TIME 1L // timeout in seconds
```
Comment (koriwi) [inline settings.hpp]: ?

Comment (legomannetje) [reply settings.hpp]: Oops. Testing the timeout function

**Pull request merged**